### DISCUSSION TITLE
How to use yolov3 without pretrained weights.

### DISCUSSION THREAD

**Discussion by Better369** [open-mmlab/mmdetection#9509] [Q&A]
### What's the feature?

When I was using the file yolov3_d53_mstrain-608_273e_coco.py, I found that I needed to use and train weights, and I also read other files of yolov3, which also required pre-training weights. Can I train without using pre-training weights?
![62a7b0a2fa7a9fed6f3c26a43992ca4](https://user-images.githubusercontent.com/109217396/208287956-d4fa9c12-ece0-4b86-a603-fd6dc73b2a04.png)

**Comment by ZwwWayne** [comment]
Yes, you can. Simply commenting the init_cfg in the config of backbone should work.

**Comment by Better369** [comment]
But when I am training, the value of loss cannot be displayed.
![03928aaa2578200669396616130417a](https://user-images.githubusercontent.com/109217396/208342899-088daa50-6f43-4a2a-863b-bba1f7c5b0f2.png)
